### PR TITLE
use modular imports for aws-sdk s3 package

### PR DESCRIPTION
**Pull request by JStein92**
See https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/index.html for docs

### CODE REVIEW

File: services/aws-s3.service.ts [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+  GetObjectCommand,
+  GetObjectCommandInput,
+} from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
 import { AppCredentialsService } from "./credentials.service";
 
 export class AppAwsS3Service {
   readonly bucketRegex = /http.?:\/\/([-\w]+).*.com/;
   readonly keyRegex = /s3.amazonaws.com\/(.*)/;
+  readonly defaultRegion = "us-west-2";
```
Comment (Morkos) [inline services/aws-s3.service.ts:13]: spoke offline, but otr-service and otr-service-devo buckets are on us-west-2 so makes to default here

File: services/aws-s3.service.ts [hunk 2]
```diff
@@ -48,37 +56,47 @@ export class AppAwsS3Service {
...
+    const keyMatches = this.keyRegex.exec(imageUrl) ?? [""];
+    const keyName = keyMatches[1];
 
-    const params = {
+    const params: GetObjectCommandInput = {
       Bucket: bucketName,
       Key: keyName,
     };
 
-    this.s3Client.getSignedUrl("getObject", params, (err, url) => {
```
Comment (Morkos) [inline services/aws-s3.service.ts:74]: +1 on using no more call back fxn here

File: services/aws-s3.service.ts [hunk 3]
```diff
@@ -48,37 +56,47 @@ export class AppAwsS3Service {
...
+      apiVersion: options.apiVersion ?? this.defaultApiVersion,
+      credentials: awsCreds,
+      region: options.region ?? this.defaultRegion,
     });
-    this.s3Client = new AWS.S3({ credentials: awsCreds });
+    return this.s3Client;
   }
 
-  public getSignedUrlInternal(options) {
+  public async getSignedUrlInternal(options) {
```
Comment (Morkos) [inline services/aws-s3.service.ts:77]: I believe this is supposed to be a private function? Can you double check this?

**Review by Morkos — APPROVED** [review]
Conditional approval. Nice find on modular imports!

File: services/aws-s3.service.ts (hunk 3 above)
Comment (JStein92) [reply services/aws-s3.service.ts:77]: it's public for testing purposes only it seems

**Pull request merged**